src/main/java/uk/gov/pay/products/model/GatewayAccountRequest.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.pay.products.model;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+import static com.google.common.collect.Lists.newArrayList;
+import static java.lang.String.format;
+import static java.util.stream.Collectors.toList;
+import static org.apache.commons.lang3.StringUtils.isEmpty;
+
+public class GatewayAccountRequest {
```

Comment:
These PATCH requests classes look identical. They all have the same 3 fields and same sort of operations. Wonder if we can generify them so that any future PATCH'es will become much more simpler

Reply:
👍